tools/install.sh

```diff
@@ -81,33 +81,24 @@ elif [[ -x ../composer ]]; then
     echo "PHP Composer appears to be installed."
     composer_scr="./composer install --no-dev"
 else
-    echo "PHP Composer does not appear to be installed. Attempting to install now..."
-    curl -sS https://getcomposer.org/installer | php
-    mv composer.phar ../composer
-    if [[ -x ../composer ]]; then
-        echo ""
-        echo "PHP Composer successfully installed."
-        composer_scr="./composer install --no-dev"
-    else
-        echo ""
-        echo "PHP Composer failed to install. Aborting."
-        exit 2;
-    fi
+    echo ""
+    echo "PHP Composer does not appear to be installed. Please install it before running this script."
```

Comment:
Could you also echo the instructions on how to do it? i.e. `curl -sS https://getcomposer.org/installer | php`